pom.xml

```diff
@@ -310,11 +310,13 @@
         <artifactId>activemq-all</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <!-- Remove activemq-camel
       <dependency>
         <groupId>org.apache.activemq</groupId>
         <artifactId>activemq-camel</artifactId>
         <version>${project.version}</version>
       </dependency>
+      -->
```

Comment:
Likewise with these, probably just removal is clearer.

I know this might just be a first "revision", but just in case you forget

Reply:
Agreed